shoop/core/models/product_media.py

```diff
@@ -40,7 +40,10 @@ class ProductMedia(TranslatableModel):
         ProductMediaKind, db_index=True, default=ProductMediaKind.GENERIC_FILE, verbose_name=_('kind')
     )
     file = FilerFileField(blank=True, null=True, verbose_name=_('file'))
-    external_url = models.URLField(blank=True, null=True, verbose_name=u'URL')
+    external_url = models.URLField(
+        blank=True, null=True, verbose_name=u'URL',
+        help_text=_("Enter URL to external file. If this field is filled, the selected media doesn't apply.")
+    )
```

Comment:
This help text and the validation that both url and file are not set is very good improvement. Good work! :)